### PR TITLE
Add tests for SDIV instructions

### PR DESCRIPTION
**Pull request by dthaler**
Addresses part of #152

Also:
* Add libbpf-dev to ubuntu prerequisites
* Treat "Plugin returned error code" as ERROR rather than FAIL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Linux Kernel is treated as the authorative eBPF implementation.
 ### Ubuntu
 
 ```
-sudo apt-get install -y libboost-dev libboost-filesystem-dev libboost-program-options-dev libelf-dev lcov
+sudo apt-get install -y libboost-dev libboost-filesystem-dev libboost-program-options-dev libelf-dev lcov libbpf-dev
```
Comment (Alan-Jowett) [inline README.md:25]: What is this needed for? I think we download and build libbpf v1.3.0, so why would we need the old version provided by Ubuntu?

Comment (dthaler) [reply README.md:25]: There's nothing in the list of prerequisites for users that tell them to download and build libbpf.
Should there be?

Comment (Alan-Jowett) [reply README.md:25]: Got it. This is the README not the CI/CD. My bad I was confused.

File: tests/sdiv32-by-zero-reg.data [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Big Switch Networks, Inc
```
Comment (Alan-Jowett) [inline tests/sdiv32-by-zero-reg.data:1]: Are these new tests or derivative? Just curious about the license attribution.

**Review by Alan-Jowett — APPROVED** [review]

**Pull request merged**